server/src/packages/pas-form/applicants/applicants.controller.ts

```diff
@@ -39,24 +61,37 @@ export class ApplicantsController {
     return {
       dcp_applicantinformationid: id,
       ...body,
-    }
+    };
   }
 
   @Post('/')
   create(@Body() body) {
-    const allowedAttrs = pick(body, APPLICANT_ATTRIBUTES);
+    const { target_entity } = body;
+  
+    let allowedAttrs;
+
+    // determine the appropraite attributes based on the entity type
+    if (target_entity === 'dcp_applicantinformation') {
+      allowedAttrs = pick(body, APPLICANT_ATTRIBUTES);
+    } else if (target_entity === 'dcp_applicantrepinformation') {
+      allowedAttrs = pick(body, APPLICANT_REPRESENTATIVE_ATTRIBUTES);
+    }
 
     if (body.pas_form) {
-      return this.crmService.create('dcp_applicantinformations', {
+      let applicantBody = {
         ...allowedAttrs,
+      }
+      const objKey = `dcp_${target_entity}_dcp_pasform@odata.bind` 
+      // Dy365 syntax for associating a newly-created record
+      // with an existing record.
+      // see: https://docs.microsoft.com/en-us/powerapps/developer/common-data-service/webapi/create-entity-web-api#associate-entity-records-on-create
+      applicantBody[objKey] = [`/dcp_pasforms(${body.pas_form})`]
 
-        // Dy365 syntax for associating a newly-created record
-        // with an existing record.
-        // see: https://docs.microsoft.com/en-us/powerapps/developer/common-data-service/webapi/create-entity-web-api#associate-entity-records-on-create
-        'dcp_dcp_applicantinformation_dcp_pasform@odata.bind': [`/dcp_pasforms(${body.pas_form})`],
-      });
+      console.log(applicantBody);
+      
+      return this.crmService.create(`${target_entity}s`, applicantBody);
     } else {
-      return this.crmService.create('dcp_applicantinformations', allowedAttrs);
```

Comment:
@allthesignals is pluralizing the entity part of the web crm api?  Getting an error when trying to parametrize this `create` call to accomodate the `dcp_applicantrepinformation` entity.